src/facets/erc20-mintable/ERC20MintableFacet.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.19;
+
+import { Facet } from "src/facets/Facet.sol";
+import { IERC20Mintable } from "./IERC20Mintable.sol";
```

Comment:
Your local imports are relative but the external ones absolute. Maybe it would be better to have one pattern?

Reply:
I think this should be done in a separate PR, since I might have previous code that is also relevant. I created #48 for this